### ISSUE TITLE
swiping through page with VO should not modify radio button selection

### ISSUE THREAD

**Issue by jnurthen** [adobe/spectrum-web-components#766]
### Expected Behaviour
I should be able to swipe through the page without changing any entries. This is the way to read a page

### Actual Behaviour
When swiping left/right on radio buttons selection changes to the focused radio button

#### Steps to Reproduce
Start VO on iOS or MacOS 
https://opensource.adobe.com/spectrum-web-components/components/radio/examples
move to the first radio button 
on IOS swipe right to move to the next - note that selection changes
on MacOS press ctrl+option right arrow to move to the next and note selection changes.

#### Platform and Version
MacOS + VO + Safari
iOS13/14 + VO + Safari

**Comment by Westbrook** [comment]
I'll look into this, but do you know the equivalent event to the swiping you are doing here? I've outlined this to work like the `name` attribute of the `<input type="radio'/>` element where focusing on the elements alters the value of the group of elements outline by a shared `name` attribute. Maybe there's something special that the arrow key based focus in this context is doing something different than actually "focusing" the element.

**Comment by Westbrook** [comment]
Clarify the difference between "focus" and "reader focus", then rely on "click" only in the reader case.
